docs/01-readme.md

```diff
@@ -130,9 +130,7 @@ for any coding, documentation or patch guidelines.
 
 ## Documentation
 
-Online documentation is available at http://awesome.naquadah.org/doc/ for the
-stable branch and at http://awesomewm.github.io/apidoc/ for the master branch.
-It can be built using `make ldoc`.
+Online documentation is available at http://awesome.org/apidoc/.
```

Comment:
Currently we have both `/doc` (part of the awesome-www repo and build via Travis) and `/apidoc` (via the `apidoc` repo, always points to master).

Since this file is the README both for our GitHub repo (should point to `/apidoc`, I guess) and for the built documentation that gets installed (shouldn't point at any web pages at all, since the docs are already available), I'm not really sure what to do here.
Thoughts?